initializr-generator-spring/src/test/java/io/spring/initializr/generator/spring/container/dockercompose/DockerComposeServiceFixtures.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2012-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.initializr.generator.spring.container.dockercompose;
+
+import java.util.Collections;
+
+/**
+ * Fixtures for {@link DockerComposeService}.
+ *
+ * @author Moritz Halbritter
+ */
+final class DockerComposeServiceFixtures {
```

Comment:
I have nothing against this but `Fixture` is the first occurrence in the project so wondering if it should be named like that. There are a lot of such helper class across the code base.